backend/src/api/configs/error.config.ts

```diff
@@ -3,14 +3,14 @@
  * This is useful for organzing and centralizing all the error messages in one place.
  */
 
-export const AuthError = {
-    USER_NOT_FOUND: 'User not found',
-    INVALID_CREDENTIALS: 'Invalid credentials',
-    USER_EXISTS: 'User with this email already exists',
+const AuthError = {
+    USER_NOT_FOUND: 'User with this email not found when logging in',
+    INVALID_CREDENTIALS: 'User with invalid credentials when logging in',
+    USER_EXISTS: 'User with same email already exists when signing up',
```

Comment:
User with "this" email instead of "same" would make it consistent with the above one

Reply:
Will change this in next PR.